woocommerce/woocommerce-gutenberg-products-block
# Storybook and TS migration of `ExternalLinkCard` component (#5256)

This PR adds stories for the `ExternalLinkCard` component and migrates it to TypeScript.

Fixes #5256

### Testing

### Manual Testing

How to test the changes in this Pull Request:

1. Run `npm run storybook`
2. Explore the [ExternalLinkCard stories](http://localhost:6006/?path=/story/woocommerce-blocks-editor-components-externallinkcard--default). You should see one story with fully editable controls.

### User Facing Testing
These are steps for user testing (where "user" is someone interacting with this change that is not editing any code).
